adafruit_ssd1327.py

```diff
@@ -53,9 +53,14 @@
 
 # pylint: disable=too-few-public-methods
 class SSD1327(displayio.Display):
-    """SSD1327 driver"""
+    """SSD1327 driver
+    kwargs:
```

Comment:
We use sphinx style docstrings to denote arguments:  https://docs.circuitpython.org/en/latest/docs/design_guide.html#param-type

See the other libraries' docstrings for how it should look.  I'll take another look afterwards :)

Reply:
ok, learn guide that uses this module only discusses height & width - nothing about rotation.  I still question from code if using rotation would actually accomplish what I think of as 'rotation'

Reply:
`rotation` does not need to be provided, but can - that's what it's checking for in `if "rotation" in kwargs`.  Optional in terms of python types is misleading; it means that it can take `None` as an option, NOT that it is optional to provided.  `rotation` cannot take `None`, so `Optional` isn't the correct type.  Let's document it as `int`, though this is worth revisiting since I think it may be better to treat it as bool at some point.